docs/management/console-rack-management.md

```diff
@@ -59,3 +59,65 @@ You can move any Console-managed Rack back to being locally managed only with th
     staging            gcp       running
 ```
 Terraform state will be transferred to your local machine for exclusive management.
+## View Kubernetes resources
+You can view the Kubernetes resources deployed to your cluster with the AWS Management Console.  
+
+### Steps:
+- Create a Kubernetes clusterrolebinding that is bound to a Kubernetes clusterrole that has the necessary permissions to view the Kubernetes resources. To learn more about Kubernetes roles and role bindings, see Using RBAC Authorization in the Kubernetes documentation. You can apply one of the following manifests to your cluster that create a role and rolebinding or a clusterrole and clusterrolebinding with the necessary Kubernetes permissions: *
+If your rack version is > 3.6.4, you don’t need to create the Cluster role and the Cluster role binding.
```

Comment:
Can you move the _You can skip it for rack greater or equal to 3.6.4_ to the begninng? using Pascal case or separating the words on `clusterrolebinding` will be easier to read? (i.e. `ClusterRoleBiding` or _cluster role binding_)

Reply:
> Using RBAC Authorization in the Kubernetes documentation

Can you link the referred documentation here?